packages/esm-patient-test-results-app/translations/am.json

```diff
@@ -1,23 +1,35 @@
 {
   "backToTimeline": "Back to timeline",
+  "cancel": "Cancel",
   "date": "Date",
   "dateCollected": "Displaying date collected",
   "emptyStateText": "There are no {{displayText}} to display for this patient",
+  "endDate": "End date",
+  "female": "Female",
+  "male": "Male",
   "moreResultsAvailable": "More results available",
   "observationsDisplayText": "observations",
+  "onDate": "on",
   "ordered": "Ordered",
+  "other": "Other",
+  "print": "Print",
+  "printedBy": "Printed by",
+  "printTestResults": "Print test results",
   "recentResults": "Recent Results",
   "recentTestResults": "recent test results",
   "record": "Record",
   "referenceRange": "Reference range",
   "resetTreeText": "Reset tree",
   "resulted": "Resulted",
+  "resultsAvailableWithinTheSelectedTimeframe": " test results found within the selected time period",
   "resultsText": "results",
   "returnToTimeline": "Return to timeline",
   "seeAllResults": "See all results",
   "showTree": "Show tree",
+  "startDate": "Start date",
   "Test Results": "Test Results",
   "testName": "Test name",
+  "testReasults": "Test results",
```

Comment:
```suggestion
```

When fixing typos, please use ctrl-shift-F global search to clean up. A global search of "testReasults" would have found this.